src/main/scala/com/haufe/umantis/ds/nlp/LinearWeigher.scala

```diff
@@ -45,7 +45,13 @@ class LinearWeigher(override val uid: String)
 
     val linearCombination = udf {
       (linearWeights:Row, values: Seq[Float]) => {
-        (values, linearWeights.getSeq[Float](1)).zipped.map(_ * _).sum / linearWeights.getFloat(0)
```

Comment:
So what is the problem here? that linearWeights is shorter than values? In that case I think it should be addressed _before_ calling the udf as it might drive you to believe this is behaving correctly when it's not.

Reply:
Problem is that i want ignore null columns and also remove weights of it